LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, Max Planck Institute for Intelligent Systems
```

Comment:
The copyright should be "Max Planck Gesellschaft"

Reply:
changed to Max-Planck-Gesellschaft.

Reply:
is BSD 3 ok?

Reply:
yep